### PR TITLE
fix: White screen when no image fetched from server.

### LINKED ISSUE #1159: Images are missing
**Describe the bug**
Images are missing from events and event detail section.

<p align="center">
 
 
</p>

**Would you like to work on the issue?**
- [x] No

### PR DESCRIPTION
**Pull request by angmas1**
Fixes #1159 

Changes: Showed event image which is shown on eventyay website when no image is attached to an event in all itemViews which showed white image before.

Screenshots for the change:
image

### CODE REVIEW

File: app/src/main/res/layout/content_event.xml [hunk 1]
```diff
@@ -23,7 +23,7 @@
             app:layout_constraintHorizontal_bias="0.5"
             app:layout_constraintStart_toStartOf="parent"
             app:layout_constraintTop_toTopOf="parent"
-            tools:srcCompat="@drawable/placeholder" />
```
Comment (nikit19) [inline app/src/main/res/layout/content_event.xml]: there is no need of removing this line. This to display the placeholder in the design screen

File: app/src/main/java/org/fossasia/openevent/general/event/EventDetailsFragment.kt [hunk 2]
```diff
@@ -228,11 +228,13 @@ class EventDetailsFragment : Fragment() {
         }
 
         // Set Cover Image
-        event.originalImageUrl?.let {
-            Picasso.get()
+        event.originalImageUrl.let {
+            if (it != null) {
```
Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/openevent/general/event/EventDetailsFragment.kt]: Removing this additional check is the entire point of using ?.let

Comment (angmas1) [reply app/src/main/java/org/fossasia/openevent/general/event/EventDetailsFragment.kt]: Re-adding

Comment (angmas1) [reply app/src/main/java/org/fossasia/openevent/general/event/EventDetailsFragment.kt]: Resolved

**Comment by angmas1** [comment]
@iamareebjamal can you specify which lines are unrelated.

File: app/src/main/java/org/fossasia/openevent/general/event/EventDetailsFragment.kt [hunk 3]
```diff
@@ -230,9 +230,9 @@ class EventDetailsFragment : Fragment() {
         // Set Cover Image
         event.originalImageUrl?.let {
             Picasso.get()
-                    .load(it)
-                    .placeholder(R.drawable.ic_launcher_background)
-                    .into(rootView.logo)
```
Comment (liveHarshit) [inline app/src/main/java/org/fossasia/openevent/general/event/EventDetailsFragment.kt]: Why these changes?

File: app/src/main/java/org/fossasia/openevent/general/event/EventViewHolder.kt [hunk 4]
```diff
@@ -32,9 +32,9 @@ class EventViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
         setFabBackground(event.favorite)
         event.originalImageUrl?.let {
             Picasso.get()
-                    .load(it)
-                    .placeholder(R.drawable.ic_launcher_background)
-                    .into(itemView.eventImage)
```
Comment (liveHarshit) [inline app/src/main/java/org/fossasia/openevent/general/event/EventViewHolder.kt]: same

File: app/src/main/java/org/fossasia/openevent/general/order/OrdersViewHolder.kt [hunk 5]
```diff
@@ -37,9 +37,9 @@ class OrdersViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
 
         event.originalImageUrl?.let {
             Picasso.get()
-                    .load(it)
-                    .placeholder(R.drawable.ic_launcher_background)
-                    .into(itemView.eventImage)
+                .load(it)
+                .placeholder(R.drawable.ic_launcher_background)
+                .into(itemView.eventImage)
```
Comment (liveHarshit) [inline app/src/main/java/org/fossasia/openevent/general/order/OrdersViewHolder.kt]: same

File: app/src/main/java/org/fossasia/openevent/general/event/EventDetailsFragment.kt (hunk 3 above)
Comment (angmas1) [reply app/src/main/java/org/fossasia/openevent/general/event/EventDetailsFragment.kt]: reverted

File: app/src/main/java/org/fossasia/openevent/general/event/EventViewHolder.kt (hunk 4 above)
Comment (angmas1) [reply app/src/main/java/org/fossasia/openevent/general/event/EventViewHolder.kt]: reverted

File: app/src/main/java/org/fossasia/openevent/general/order/OrdersViewHolder.kt (hunk 5 above)
Comment (angmas1) [reply app/src/main/java/org/fossasia/openevent/general/order/OrdersViewHolder.kt]: reverted

**Comment by angmas1** [comment]
@liveHarshit please review
